documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -823,7 +823,11 @@ The connector adds missing columns to the table by comparing the incoming event'
 |[[jdbc-property-table-name-format]]<<jdbc-property-table-name-format, `+table.name.format+`>>
 |`+${topic}+`
 |Specifies a string that determines how the destination table name is formatted, based on the topic name of the event.
-The placeholder, `+${topic}+`, is replaced by the topic name.
+The placeholder, `+${topic}+`, is replaced by the topic name. +
+ +
+This configuration can also be specified using the `+${source._field_}+` pattern to use values from the Debezium change event's `source` information block for more dynamic formats.
```

Comment:
```suggestion
You can also use this property to dynamically influence the names of target tables, based on values in incoming events.
By applying this feature of the property, you can implement dynamic naming without having to write a custom single message transformation. +
 +
To enable dynamic naming of output tables, set the value of this property to a pattern such as `+${source._field_}+`.
This type of pattern instructs the connector to extract values from the `source` block of the {prodname} change event for use in constructing the table name.
```

Reply:
I think that it makes sense to present the use case for enabling dynamic table naming via the property vs. using an SMT near the beginning of the description rather than mentioning it later. WDYT?